recipe-server/client/control/components/action_fields/RolloutFields.js

```diff
@@ -0,0 +1,78 @@
+import React, { PropTypes as pt } from 'react';
+import { connect } from 'react-redux';
+import { selector } from 'control/components/RecipeForm';
+import { ControlField } from 'control/components/Fields';
+import ActionFields from 'control/components/action_fields/ActionFields';
+
+/**
+ * Form fields for the preference-rollout action.
+ */
+export class RolloutFields extends ActionFields {
+  static propTypes = {
+    recipeArguments: pt.object.isRequired,
+    disabled: pt.bool,
+  }
+
+  render() {
+    const {
+      recipeArguments,
+      disabled,
+    } = this.props;
+
+    return (
+      <div className="arguments-fields">
+        <p className="info">
+          Change a preference for a portion of the population.
+          <br /><br />
+          Supports gradually increasing the number of users that get a feature, multiple features
+          rolling out at once to in/dependent groups, and rolling back a deployment entirely
+          (if problems arise).
```

Comment:
Gotcha. The UX part of me thinks it'd be a good idea to have a little blurb describing the action, especially now that there is Preference Rollout _and_ Preference Experiments. Perhaps it could be a short bit of copy explaining the difference between `Rollout`s and `Experiment`s?